lib/iex/lib/iex/helpers.ex

```diff
@@ -543,7 +555,10 @@ defmodule IEx.Helpers do
     case File.ls(path) do
       {:ok, items} ->
         sorted_items = Enum.sort(items)
-        ls_print(path, sorted_items)
+        print_function = fn(item, width) -> 
```

Comment:
Inside`IEx.Introspection` module (and I believe other places as well) we call such function as `printer`, would be beneficial to be consistent I think.